src/templates/next/components/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,14 @@
+import { ParagraphProps } from "~/common"
+import { getSanitizedInlineContent } from "~/utils/getSanitizedInlineContent"
+
+const Paragraph = ({ content }: ParagraphProps) => {
+  const sanitizedContent = getSanitizedInlineContent(content)
+  return (
+    <p
+      className="[&_a]:underline [&_a]:text-blue-500 hover:[&_a]:text-blue-700"
```

Comment:
can someone clue me in on this HAHA totally lost

Reply:
This sets all child elements that are of `a` tag to have underline and be blue in colour.